### PR TITLE
direnv v2.6.0

### PR DESCRIPTION
**Pull request by zimbatm**
Just a new release

### CODE REVIEW

File: Library/Formula/direnv.rb [hunk 1]
```diff
@@ -2,17 +2,11 @@
...
+  url 'https://github.com/zimbatm/direnv/archive/v2.6.0.tar.gz'
+  sha1 'ebfa8c087aadeffe2c8f84128082d670d22ea541'
 
   head 'https://github.com/zimbatm/direnv.git'
 
-  bottle do
-    sha1 "fb5340876bc488bd38aba9e00d52fff0f029f797" => :yosemite
-    sha1 "fbc2ed0cadad4543df0d09e3afe756866e0e44b3" => :mavericks
-    sha1 "353d3f25f0b7e3bd57bc91dd855635163bc426e7" => :mountain_lion
-  end
```
Comment (mikemcquaid) [inline Library/Formula/direnv.rb]: Just leave this as-is and we'll update it when we pull. Thanks!

File: Library/Formula/direnv.rb [hunk 2]
```diff
@@ -23,8 +17,8 @@ def caveats; <<-EOS.undent
     At the END of your ~/.bashrc or ~/.zshrc, add the following line:
       eval "$(direnv hook $0)"
 
-    See the wiki for docs and examples:
-      https://github.com/zimbatm/direnv/wiki/
+    See the website for more docs and shell support:
+      http://www.direnv.net
```
Comment (mikemcquaid) [inline Library/Formula/direnv.rb]: Let's just remove these two lines.

Comment (zimbatm) [inline Library/Formula/direnv.rb]: What about just `check 'man direnv' to finish the install`. The shell integration and link to the website are documented there.

File: Library/Formula/direnv.rb (hunk 1 above)
Comment (zimbatm) [inline Library/Formula/direnv.rb]: Will do thanks

File: Library/Formula/direnv.rb (hunk 2 above)
Comment (mikemcquaid) [inline Library/Formula/direnv.rb]: I don't think we need either; the website is linked from the formula.

Comment (zimbatm) [inline Library/Formula/direnv.rb]: I would like to keep some form of instruction because without the shell integration direnv is not useful.

Comment (mikemcquaid) [inline Library/Formula/direnv.rb]: Ok, can we have `Configure direnv by reading:` and linking to a specific page? If there's no specific page or just the manpage I don't think that's useful; people will look at those things anyway.

Comment (zimbatm) [inline Library/Formula/direnv.rb]: Yeah makes sense. Pushed ------v

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `direnv 2.6.0` (no `v`, apologies for extreme pedantry).

**Comment by zimbatm** [comment]
Cheers. Log format nitpick duly noted.

**Pull request closed without merging**